jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -529,12 +271,12 @@ The policy files (*.rego) present in the given policy directory (searched recurs
 ```bash
 circleci policy decide --input $PATH_TO_CONFIG $PATH_TO_POLICY_DIR
 ```
-Policies that use `data.meta...` values like `branch` or `project_id` should also provide a json file mocking those values with `--metafile $PATH_TO_JSON`
 
+Policies that use `data.meta...` values like `branch` or `project_id` should also provide a json file mocking those values with `--metafile $PATH_TO_JSON`
 
 It is recommended that users build a test suite of policy/config combinations and run them locally or in CI before pushing them to their organization's active policies.
 
-# TODO: further discuss testing recommendations (maybe a separate written section)
+#TODO: further discuss testing recommendations (maybe a separate written section)#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
This is still outstanding. We could ship without it but having it would significantly improve the customer experience around testing policies.